File: AngularIS/src/SuperGasApp/ClientApp/src/models/order.model.ts
import { Customer } from "./customer.model";

export class Order {
  static id: string;

  constructor(id?: number, annotations?: string, customerId?: string, date?: Date, customer?: Customer) {
    this.id = id;
    this.annotations = annotations;
    this.customerId = customerId;
    this.date = date;
    this.customer = customer;
  }

  public id?: number;
  public annotations?: string;
  public customerId?: string;
  public date?: Date;
  public customer?: Customer;
  public index?: number;
}

File: AngularIS/src/SuperGasApp/Model/OrderModel.cs
using SuperGasApp.Data.Models.Inventory;

namespace SuperGasApp.Model;

public class OrderModel
{
    public int Id { get; set; }
    public string? Annotations { get; set; }
    public DateTime Date { get; set; }
    public List<OrderProduct>? OrdersProducts { get; set; }
    public int CustomerId { get; set; }
    public CustomerModel? Customer { get; set; }
}
